lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -246,12 +246,90 @@ class PatternStopContents extends Component<Props, State> {
     updatePatternStops(activePattern, patternStops)
   }
 
+  _onPickupChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const newPickup: number = parseInt(evt.target.value)
+    const {activePattern, index, updatePatternStops} = this.props
+    const patternStops = [...activePattern.patternStops]
+
+    patternStops[index].pickupType = newPickup
+    this.setState({update: true})
+    updatePatternStops(activePattern, patternStops)
+  }
+
+  _onDropOffChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const newDropOff: number = parseInt(evt.target.value)
+    const {activePattern, index, updatePatternStops} = this.props
+    const patternStops = [...activePattern.patternStops]
+
+    patternStops[index].dropOffType = newDropOff
+    this.setState({update: true})
+    updatePatternStops(activePattern, patternStops)
+  }
+
+  _onContinuousPickupChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const newPickup: number = parseInt(evt.target.value)
+    const {activePattern, index, updatePatternStops} = this.props
+    const patternStops = [...activePattern.patternStops]
+
+    patternStops[index].continuousPickup = newPickup
+    this.setState({update: true})
+    updatePatternStops(activePattern, patternStops)
+  }
+
+  _onContinuousDropOffChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const newPickup: number = parseInt(evt.target.value)
+    const {activePattern, index, updatePatternStops} = this.props
+    const patternStops = [...activePattern.patternStops]
+
+    patternStops[index].continuousDropOff = newPickup
```

Comment:
Rename `newPickup` to `newDropOff` or something to match the attribute being changed.